src/args.rs

```diff
@@ -0,0 +1,29 @@
+use clap:: {
+    Parser,
+};
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+pub struct PokerArgs{
+    // Option command to run the program
+    #[arg(short = 'r', long = "run")]
+    /// Run program
+    pub run_option: bool,
+    
+    // Option command to display the deck of card
+    #[arg(short = 'd', long = "display")]
+    /// Run program
+    pub display_option: bool,
+
+    // Optional argument to get the number of cards for playing
+    // Set to default so that if one doesn't any value it goes to demo mode
+    #[arg(required = false, default_value_t = 0, requires = "run_option")]
```

Comment:
This `requires` one is interesting. Neat.  That will be useful as the API gets build out.